### PR TITLE
LPS-148033 form filed project should to update node_modules path depends on node package manager type

### PR DESCRIPTION
**Pull request by simonjhy**

### CODE REVIEW

**Comment by simonjhy** [comment]
ci:test:project-templates

**Comment by simonjhy** [comment]
ci:test:project-templates

**Comment by simonjhy** [comment]
ci:test:project-templates

**Comment by simonjhy** [comment]
ci:test:sf

**Comment by simonjhy** [comment]
ci:test:sf

**Comment by simonjhy** [comment]
ci:test:sf

**Comment by simonjhy** [comment]
ci:test:sf

**Comment by simonjhy** [comment]
ci:test:project-templates

File: modules/sdk/project-templates/project-templates-form-field/src/main/java/com/liferay/project/templates/form/field/internal/FormFieldProjectTemplateCustomizer.java [hunk 1]
```diff
@@ -83,6 +92,67 @@ public void onAfterGenerateProject(
...
+					gradlePropertiesPath, StandardOpenOption.READ)) {
+
+				Properties gradleProperties = new Properties();
+
+				gradleProperties.load(gradlePropertiesInputStream);
+
+				String nodeManager = gradleProperties.getProperty(
+					"liferay.workspace.node.package.manager");
+
+				if (StringUtils.equals(nodeManager, "yarn")) {
```
Comment (lawrence-lee) [inline modules/sdk/project-templates/project-templates-form-field/src/main/java/com/liferay/project/templates/form/field/internal/FormFieldProjectTemplateCustomizer.java:125]: @simonjhy , Did you test this locally?

Comment (simonjhy) [reply modules/sdk/project-templates/project-templates-form-field/src/main/java/com/liferay/project/templates/form/field/internal/FormFieldProjectTemplateCustomizer.java:125]: yes

Comment (simonjhy) [reply modules/sdk/project-templates/project-templates-form-field/src/main/java/com/liferay/project/templates/form/field/internal/FormFieldProjectTemplateCustomizer.java:125]: I will fix it.

**Comment by simonjhy** [comment]
ci:test:sf

**Comment by simonjhy** [comment]
ci:test:project-templates

**Pull request closed without merging**